cli/standalone/mod.rs

```diff
@@ -495,6 +501,46 @@ impl ModuleLoader for EmbeddedModuleLoader {
     }
     std::future::ready(()).boxed_local()
   }
+
+  fn get_source_map(&self, file_name: &str) -> Option<Vec<u8>> {
+    if file_name.starts_with("file:///") {
+      let url =
+        deno_path_util::url_from_directory_path(self.shared.vfs.root()).ok()?;
+      let file_url = ModuleSpecifier::parse(file_name).ok()?;
+      let relative_path = url.make_relative(&file_url)?;
+      self.shared.source_maps.get(&relative_path)
+    } else {
+      self.shared.source_maps.get(file_name)
+    }
+    // todo(https://github.com/denoland/deno_core/pull/1007): don't clone
```

Comment:
Should we release `deno_core` before landing this PR?

Reply:
I think it's fine to not wait.